test/MorphoToken.t.sol

```diff
@@ -201,4 +201,10 @@ contract MorphoTokenTest is BaseTest {
         assertEq(newMorpho.getVotes(delegatee1), initialAmount - transferredAmount);
         assertEq(newMorpho.getVotes(delegatee2), transferredAmount);
     }
+
+    function testERC20DelegatesStorageLocation() public pure {
+        bytes32 expected =
+            keccak256(abi.encode(uint256(keccak256("morpho.storage.ERC20Delegates")) - 1)) & ~bytes32(uint256(0xff));
+        assertEq(expected, 0x1dc92b2c6e971ab6e08dfd7dcec0e9496d223ced663ba2a06543451548549500);
```

Comment:
Maybe this would be a reason to have storage locations internal instead of private, so we can test them. For storage locations defined by the lib it's fine, because they should be tested independently, but for our added definitions it could be beneficial

Reply:
Not a strong opinion on this

Reply:
done in #45